### PR TITLE
[QNEBE-611] Use email rather than username to login

### PR DESCRIPTION
**Pull request by platelminto**

### CODE REVIEW

**Comment by platelminto** [comment]
Pylint complains about many of the options we have in .pylintrc, unsure if they've always failed or not?

**Comment by QFer** [comment]
With me, it only complains about the same 2 issues why pylint (pip install pylint==2.15.3) fails in the pipeline:
src\adk\api\qne_client.py:164:17: W3101: Missing timeout argument for method 'requests.get' can cause your program to hang indefinitely (missing-timeout)
src\adk\api\qne_client.py:171:17: W3101: Missing timeout argument for method 'requests.post' can cause your program to hang indefinitely (missing-timeout)

File: src/adk/api/qne_client.py [hunk 1]
```diff
@@ -175,7 +175,7 @@ def _client_post(url: str, **params: ParametersType) -> Any:
```
Comment (QFer) [inline src/adk/api/qne_client.py:175]: Error in pipeline:

With me, it only complains about the same 2 issues why pylint (pip install pylint==2.15.3) fails in the pipeline:
src\adk\api\qne_client.py:164:17: W3101: Missing timeout argument for method 'requests.get' can cause your program to hang indefinitely (missing-timeout)
src\adk\api\qne_client.py:171:17: W3101: Missing timeout argument for method 'requests.post' can cause your program to hang indefinitely (missing-timeout)

I guess adding "timeout=10" as a parameter is oke. Waiting for 10 seconds is pretty long...

File: README.md [hunk 2]
```diff
@@ -411,12 +411,12 @@ Arguments:
   [HOST]  Name of the host to log in to
 
 Options:
-  --username TEXT  Username of the remote user  [required]
+  --email TEXT     Email of the remote user  [required]
   --password TEXT  Password of the remote user  [required]
   --help           Show this message and exit.
 
 Example:
-  qne login --username=my_user_name --password=my_password https://api.quantum-network.com
```
Comment (QFer) [inline README.md:419]: Not for this repo but we should change and possibly change the descriptions on the QNE-ADK website when we release the new adk. Maybe add a ticket for this to find and adjust all the references to username on the website

**Review by QFer — CHANGES_REQUESTED** [review]

File: src/adk/api/qne_client.py (hunk 1 above)
Comment (platelminto) [reply src/adk/api/qne_client.py:175]: Done.

**Comment by QFer** [comment]
The changes in this pull request are integrated in QNEBE-622, see https://github.com/QuTech-Delft/qne-adk/commit/1d996647efd3bbd17a0c5aa952a1c0af6c6d3310
This PR can be closed.

**Pull request closed without merging**